tools/release.sh

```diff
@@ -117,14 +107,15 @@ done
 
 mkdir "dist.$version"
 mv dist "dist.$version/josepy"
+poetry export -f requirements.txt > constraints.txt
```

Comment:
I think adding this to `.gitignore` like you suggested [here](https://github.com/certbot/josepy/pull/129#discussion_r845571073) is just fine, but it looks like that change has it made its way to this branch yet.

Reply:
Indeed the commit was pending on my local repo. It is fixed.